scripts/update_dependencies.sh

```diff
@@ -0,0 +1,40 @@
+# Pip dependency update script
+# Usage:
+# bash <path to script> [Optional branch name]
+#
+# If the branch name is specified install the requirements from that branch
+# otherwise install from develop
+set -e
+
+if [ "$1" ]
+  then
+    REQ_BRANCH="$1"
+else
+    REQ_BRANCH="develop"
+fi
+
+# make sure bluemira is in the current environment
+BM_INSTALLED=$(pip list | grep bluemira | awk 'NR==1 {print $1}')
+[ "$BM_INSTALLED" = "bluemira" ] || { echo "bluemira not found in the current python environment, exiting" && exit 1; }
+
+readonly BLUEMIRA_ROOT="$(realpath "$(dirname "$0")"/..)"
+echo 'Bluemira directory is '$BLUEMIRA_ROOT
+
+OLD_DIR=$(pwd)
+cd $BLUEMIRA_ROOT # git commands dont work when not in a git directory
+
+# check that the requirements files havent changed
+git diff --exit-code $BLUEMIRA_ROOT"/requirements.txt" || { cd $OLD_DIR && echo "requirements.txt modified on this branch, exiting" && exit 1; }
```

Comment:
These are long commands, we could split them onto multiple lines 
```suggestion
git diff --exit-code $BLUEMIRA_ROOT"/requirements.txt" \
  || { cd $OLD_DIR && echo "requirements.txt modified on this branch, exiting" && exit 1; }
```